exporter/elasticsearchexporter/bulkindexer_test.go

```diff
@@ -641,8 +82,9 @@ func TestSyncBulkIndexer_flushBytes(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			var reqCnt atomic.Int64
 			cfg := Config{
-				NumWorkers:   1,
-				Flush:        FlushSettings{Interval: time.Hour, Bytes: 1},
+				QueueBatchConfig: exporterhelper.QueueBatchConfig{
+					NumConsumers: 1,
```

Comment:
nit: specify equiv batcher config to avoid relying on default and avoid flaky test

Reply:
:facepalm: missed it. You're right